lib/option_set_creator.rb

```diff
@@ -14,200 +14,306 @@ def create_all
-      })
-    media_kind.options.create(value: 'document', label_translations: {
-        en: I18n.t('database.option_sets.media_kind.document', locale: 'en'),
-        es: I18n.t('database.option_sets.media_kind.document', locale: 'es')
-      })
-    media_kind.options.create(value: 'contract', label_translations: {
-        en: I18n.t('database.option_sets.media_kind.contract', locale: 'en'),
-        es: I18n.t('database.option_sets.media_kind.contract', locale: 'es')
-      })
+
+    image = media_kind.options.find_or_create_by(value: 'image')
+    image.label_translations = {
+      en: I18n.t('database.option_sets.media_kind.image', locale: 'en'),
+      es: I18n.t('database.option_sets.media_kind.image', locale: 'es')
+    }
+    image.save
+
+    video = media_kind.options.find_or_create_by(value: 'video')
+    video.label_translations = {
+      en: I18n.t('database.option_sets.media_kind.video', locale: 'en'),
+      es: I18n.t('database.option_sets.media_kind.video', locale: 'es')
+    }
+    video.save
+
+    document = media_kind.options.find_or_create_by(value: 'document')
+    document.label_translations = {
+      en: I18n.t('database.option_sets.media_kind.document', locale: 'en'),
+      es: I18n.t('database.option_sets.media_kind.document', locale: 'es')
+    }
+    document.save
+
+    contract = media_kind.options.find_or_create_by(value: 'contract')
+    contract.label_translations = {
+      en: I18n.t('database.option_sets.media_kind.contract', locale: 'en'),
+      es: I18n.t('database.option_sets.media_kind.contract', locale: 'es')
+    }
+    contract.save
   end
 
   def create_loan_transaction_type
     loan_transaction_type = OptionSet.find_or_create_by(division: Division.root,
       model_type: Accounting::Transaction.name, model_attribute: 'loan_transaction_type')
-    loan_transaction_type.options.destroy_all
-    loan_transaction_type.options.create(value: 'interest', position: 1, label_translations: {
-        en: I18n.t('database.option_sets.loan_transaction_type.interest', locale: 'en'),
-        es: I18n.t('database.option_sets.loan_transaction_type.interest', locale: 'es'),
-      })
-    loan_transaction_type.options.create(value: 'disbursement', position: 2, label_translations: {
-        en: I18n.t('database.option_sets.loan_transaction_type.disbursement', locale: 'en'),
-        es: I18n.t('database.option_sets.loan_transaction_type.disbursement', locale: 'es')
-      })
-    loan_transaction_type.options.create(value: 'repayment', position: 3, label_translations: {
-        en: I18n.t('database.option_sets.loan_transaction_type.repayment', locale: 'en'),
-        es: I18n.t('database.option_sets.loan_transaction_type.repayment', locale: 'es')
-      })
-    loan_transaction_type.options.create(value: 'other', position: 4, label_translations: {
-        en: 'Other',
-        es: 'Otro'
-      })
+
+    interest = loan_transaction_type.options.find_or_create_by(value: 'interest', position: 1)
```

Comment:
Do you know why we use position in this method?

Reply:
I think because the options are intended to have an order other than alphabetical, so it's needed for sorting purposes, but I'd probably rather set those values only in the `find_or_create` and update the position with the translations...